src/updater.ts

```diff
@@ -0,0 +1,131 @@
+import * as k8s from "@kubernetes/client-node";
+import type { KubeConfig } from "@kubernetes/client-node";
+import * as yaml from "js-yaml";
```

Comment:
I think `@kubernetes/client-node` exposes yaml helper -> no need for additional dependency?

Reply:
Thanks, fixed. I noticed that `@kubernetes/client-node` uses the same dependency, but it's better this way.